employees/forms.py

```diff
@@ -113,11 +107,10 @@ def _set_last_choices_in_report_form(self, author: CustomUser) -> None:
                 author in last_report.project.members.all() or author in last_report.project.managers.all()
             ):
                 self.initial["project"] = last_report.project
-                self.fields["task_activities"].queryset = last_report.project.project_activities.all()
                 self.initial["task_activities"] = last_report.task_activities
             elif self.fields["project"].queryset.exists():
                 self.initial["project"] = self.fields["project"].queryset.first()
-                self.fields["task_activities"].queryset = self.initial["project"].project_activities.all()
+
```

Comment:
Not needed space in bigger context. You just deleted from this line code but left blank line

Reply:
changed